app/scripts/services/pmx-converter.js

```diff
@@ -10,20 +10,20 @@
   function PMXConverter(jsyaml) {
 
     function convert(pmxYaml) {
-      var pmxJson, images, newJson;
+      var pmxJson,
+        images,
+        newJson = {};
 
       try {
         pmxJson = jsyaml.safeLoad(pmxYaml);
-      } catch (YamlException) {
-        return null;
+        images = pmxJson.images;
+        images.forEach(function (image) {
+          newJson[image.name.replace(' ', '')] = imageToServiceDefinition(image);
```

Comment:
So my recent hating on regexes is gonna make a hypocrite out of me, but I'll blame javascripts replace method instead... In this case you wouldn't catch multiple spaces:
```
'foo  bar'.replace(' ', '') >> "foo bar" // only replaces one space
'foo  bar'.replace(/\s+/, '') >> "foobar" // works
'foo  bar'.replace(/\s/g, '') >> "foobar" // also works
```